android/app/src/main/java/updated/mysterium/vpn/ui/manual/connect/filter/FilterAdapter.kt

```diff
@@ -0,0 +1,119 @@
+package updated.mysterium.vpn.ui.manual.connect.filter
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.core.content.ContextCompat
+import androidx.core.content.res.ResourcesCompat
+import androidx.recyclerview.widget.RecyclerView
+import kotlinx.android.synthetic.main.item_node.view.*
```

Comment:
Not urgent, but start to use binding for recyclerView instead of synthetic for views. It's better to start now, because they will not be supported from August 2021. In last version of Kotlin synthetics are deprecated

Reply:
Fix